src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -74,8 +74,9 @@
 
 # BEWARE: this import makes it impossible to instantiate this CE client side
 from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB import PilotAgentsDB
+from DIRAC.WorkloadManagementSystem.Client import PilotStatus
 
-from DIRAC.WorkloadManagementSystem.Agent.SiteDirector import WAITING_PILOT_STATUS
+from DIRAC.WorkloadManagementSystem.Agent.SiteDirector import PILOT_WAITING_STATES
```

Comment:
probably needs a different module to import from

Reply:
Oops, I forgot that. Thanks